react/src/loginForm.js

```diff
@@ -0,0 +1,76 @@
+import { Form, Input, Button, Checkbox } from 'antd';
+import React from 'react';
+
+class LoginForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.submitFormForLogin = this.submitFormForLogin.bind(this);
+  }
+
+  onFinish(values) {
+    console.log('Success i', values);
+  }
+
+  onFinishFailed(errorInfo) {
+    console.log('Failed:', errorInfo);
+  }
+
+  submitFormForLogin(values) {
```

Comment:
I have asked you to get the correct values for username and password printed in the log. Unfortunately, I don't think your current code will do that.